packages/terra-list/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## 4.63.0 - (September 26, 2023)
 
 * Added
+  * Added arrow key navigation support for list items.
```

Comment:
this should be moved to unreleased section !!

Reply:
Added thanks